changes-set.txt

```diff
@@ -89,6 +89,7 @@ Date      Old       New         Notes
 31-Jan-25 ---       ---         Moved surreal cofinality theorems
                                 from SF's mathbox to main set.mm
 30-Jan-25 syl5bir   biimtrrid
+28-Jan-25 elreimasng ---        Deleted from iset.mm; use elrelimasn instead
```

Comment:
Do we really track such changes in iset.mm here? I did not find any entry for iset.mm here until now.

Reply:
The reason that a list for set.mm to some extent works for iset.mm is that iset.mm follows set.mm naming. So I guess that's why I'm not completely sure what to do.

But I guess you are right - it is a lot simpler to say that `changes-set.txt` is for set.mm. We do, after all, have git logs and other ways of noting changes to iset.mm.

I'll remove this entry.